packages/rum-angular/package.json

```diff
@@ -4,46 +4,35 @@
   "description": "Elastic APM Real User Monitoring for Angular applications",
   "homepage": "https://www.elastic.co/guide/en/apm/agent/rum-js/current/index.html",
   "license": "MIT",
-  "main": "dist/lib/index.js",
-  "module": "dist/es/index.js",
-  "es2015": "dist/es2015/index.js",
-  "source": "src/index.ts",
-  "files": [
-    "dist",
-    "src"
-  ],
   "publishConfig": {
-    "access": "public"
+    "access": "public",
+    "directory": "dist"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/elastic/apm-agent-rum-js.git",
     "directory": "packages/rum-angular"
   },
   "engines": {
-    "node": ">=8.0.0"
+    "node": ">=12.0.0"
   },
   "scripts": {
-    "prepublishOnly": "npm run build",
-    "build": "run-p build:main build:module build:es2015",
-    "build:main": "BABEL_ENV=BROWSER_PROD npx babel -x '.ts' src -d dist/lib ",
-    "build:module": "BABEL_ENV=BROWSER_ESM_PROD npx babel -x '.ts' src -d dist/es",
-    "build:es2015": "BABEL_ENV=BROWSER_ESM_ES2015 npx babel -x '.ts' src -d dist/es2015",
-    "build:e2e": "npm run script buildE2eBundles packages/rum-angular/test/e2e",
-    "test:unit": "npm run script runUnitTests packages/rum-angular",
-    "test:e2e": "npm run script runE2eTests packages/rum-angular/wdio.conf.js",
-    "test:sauce": "npm run script runSauceTests packages/rum-angular true test:unit build:e2e test:e2e",
-    "test": "npm run test:sauce",
-    "script": "node ../../dev-utils/run-script.js",
-    "karma:dev": "karma start --auto-watch --restartOnFileChange",
-    "karma:coverage": "karma start --coverage --singleRun"
+    "build": "ng build --prod",
+    "test:unit": "ng test",
+    "test": "MODE=none npm run test:unit",
```

Comment:
What is currently the issue with running in saucelabs?

Reply:
basically the build system has to migrated from `babel` -> `ng`. 

The test framework needs to be updated to https://www.protractortest.org which is the recommended way of testing Angular projects. Will be done in separate PR.